raiden/network/rpc/client.py

```diff
@@ -346,12 +344,6 @@ def monkey_patch_web3(web3, gas_price_strategy):
         # set gas price strategy
         web3.eth.setGasPriceStrategy(gas_price_strategy)
 
-        # In the version of web3.py we are using the http_retry_request_middleware
-        # is not on by default. But in recent ones it is. This solves some random
-        # crashes that happen on the mainnet as reported in issue
-        # https://github.com/raiden-network/raiden/issues/3558
```

Comment:
Removing this sounds fine for me as we have the partial syncing now. But we should keep an eye if this bites us.